_includes/adops/adops-gam-setup.html

```diff
@@ -104,6 +107,24 @@ <h2 id="step-2-add-a-creative">Step 2. Add a Creative</h2>
 &lt;/script&gt;
 </code></pre></div></div>
 
+<p>Code provided above will work for all formats:</p>
+<ul>
+  <li>amp</li>
+  <li>banner</li>
+  <li>mobile</li>
+  <li>native</li>
+  <li>video (outstream video)</li>
+</ul>
+<p>Based on provided targeting <b>hb_format</b>, GAM will load different .js library for different ad format.  
+If you want to run only banner ads than you can put <b>banner.js</b> in the first line of your gam creative, like:</p>
+
+<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>&lt;script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/banner.js"&gt;&lt;/script&gt;
+  &lt;script&gt;
+  &lt;/script&gt;
+</code></pre></div></div>
+
+<p>But, if you plan to run multiple different formats on your page, then you should to create GAM creatives like described above with script path ending with <b>%%PATTERN:hb_format%%</b></p>
+
 <p><img src="/assets/images/demo-setup/new-creative.png" alt="New creative" class="pb-lg-img" /></p>
 
 <p>Make sure the creative size is set to 1x1.  This allows us to set up size override, which allows this creative to serve on all inventory sizes.</p>
```

Comment:
Edit: 
Ensure the creative size is set to 1x1. This will enable a size override to be set up. A size override enables the creative to serve on all inventory sizes.

Reply:
should be fixed